### DISCUSSION TITLE
Who's actually using ejabberd?

### DISCUSSION THREAD

**Discussion by aqeelat** [processone/ejabberd#3876] [General]
Really impressive project and community.

I'm just wondering what are the actual use cases for the project?
Are there projects build on top of ejabberd that we might've heard of?

Sorry if this a a rude post, but I'm genuinely curious.

**Comment by licaon-kter** [comment]
ejabberd.im blog posts should answer that.

Also https://compliance.conversations.im/old if you look for ejabberd ;)

**Comment by badlop** [comment]
Years ago, such cases were tracked in [ejabberd's wikipedia](https://en.wikipedia.org/wiki/Ejabberd#Notable_deployments).

Some were announced in the ["ejabberd" tag of ProcessOne blog](https://www.process-one.net/blog/category/ejabberd/), like [EVE Online](https://www.process-one.net/blog/eve-online-chat-is-moving-to-ejabberd/) and [Nintendo Switch SPNS](https://www.process-one.net/blog/ejabberd-nintendo-switch-npns/). But nowadays, those cases are added to [Customer Stories](https://www.process-one.net/en/customers/). There you can get a lot of usercases ideas.

This is only about announced deployments, or discovered because they offer a connection method from outside and have mod_version enabled. Who knows how many deployments are used behind the scenes with custom web frontends, or as backend to other apps...
